scripts/docgen/docgen.ts

```diff
@@ -101,4 +100,61 @@ async function generateDocs(forDevsite: boolean = false) {
     [command, 'markdown', '--input', 'temp', '--output', outputFolder],
     { stdio: 'inherit' }
   );
+
+  moveRulesUnitTestingDocs(outputFolder, command);
+}
+
+async function moveRulesUnitTestingDocs(
+  mainDocsFolder: string,
+  command: string
+) {
+  const rulesOutputFolder = `${projectRoot}/docs-rut`;
+
+  // Generate from rules-unit-testing.api.json only, to get an index.md.
```

Comment:
I don't think we need this index file since it will only contain one package. The `rules-unit-testing.md` can be used as the entry point directly.

Reply:
It's possible that the top level file needs to be named index.md for devsite, so I will check about that. If so, I can rename rules-unit-testing.md (and all links to it).

Reply:
So it turns out the way the devsite files are structured, it also doesn't need the index.md, so just removed this.